compute_endpoint/globus_compute_endpoint/endpoint/interchange.py

```diff
@@ -452,9 +459,14 @@ def process_pending_results():
                             "   Marking task as failed."
                         )
 
+                        res_partial = packed_result[:256]
```

Comment:
I'm surprised we don't have a util function to shorten strings in the SDK.  I wrote a version of this in the action provider and remember your comment on it as well.  Though this is a bytes version not for str

Reply:
This is a good point!  But per our conversation, also no longer necessary for this PR.  (Relevant section completely removed!)